src/Router.tsx

```diff
@@ -1,16 +1,19 @@
-import React from 'react';
+import React, { lazy, Suspense } from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
 
-import Signin from './components/signin/signin';
 import { APP_ROUTES } from './enums';
-import Home from './pages/home/home';
+
+const Signin = lazy(() => import('./components/signin/signin'));
```

Comment:
create new file `sign-in-lazy.tsx` in same folder where component, where you will export lazy loaded component